### PR TITLE
Update and enable tests disabled in 854b7b8ab1

### LINKED ISSUE #2841: [Tests] Re-enable tests disabled in 854b7b8ab1
Tests:

* lgc/test/BuiltIns/cs-globalinvocationid.lgc
* lgc/test/BuiltIns/cs-localinvocationindex.lgc

See 854b7b8ab1 and #2596.
For context, see the discussion in #2835.

### PR DESCRIPTION
**Pull request by mbrkusanin**
Resolves #2841

### CODE REVIEW

File: lgc/test/BuiltIns/cs-globalinvocationid.lgc [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ; NOTE: Assertions have been autogenerated by tool/update_llpc_test_checks.py UTC_ARGS: --tool lgc
 ; RUN: lgc -mcpu=gfx1010 - < %s | FileCheck --check-prefixes=GFX8 %s
```
Comment (Flakebi) [inline lgc/test/BuiltIns/cs-globalinvocationid.lgc:2]: This fooled me by checking for GFX8 while it actually compiled with gfx10.1

**Review by Flakebi — APPROVED** [review]

**Comment by amdvlk-admin** [comment]
### Test summary for commit 4b54d747fcbc5330ba9d4ac5c3527b78476eb583
<details><summary>CTS tests (Failed: 0/138378)</summary><ul><li>Built with version 1.3.5.2</li>
<details open><summary>Ubuntu navi3x, Srdcvk</summary><ul>

<li>Passed: 35162/69163 (50.8%)</li>
<li>Failed: 0/69163 (0.0%)</li>
<li>Not Supported: 34001/69163 (49.2%)</li><li>Warnings: 0/69163 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu navi2x, Srdcvk</summary><ul>

<li>Passed: 35242/69215 (50.9%)</li>
<li>Failed: 0/69215 (0.0%)</li>
<li>Not Supported: 33973/69215 (49.1%)</li><li>Warnings: 0/69215 (0.0%)</li>
</ul></details>


</ul></details>

File: lgc/test/BuiltIns/cs-globalinvocationid.lgc (hunk 1 above)
Comment (jasilvanus) [reply lgc/test/BuiltIns/cs-globalinvocationid.lgc:2]: Apparently this was not correctly updated in aff7949fdd.
There seem to be more affected tests, I guess we should also fix them if we fix this one.

Comment (mbrkusanin) [reply lgc/test/BuiltIns/cs-globalinvocationid.lgc:2]: https://github.com/GPUOpen-Drivers/llpc/pull/2848

**Pull request merged**